docs/modules/deployment/pages/upgrade/diff.adoc

```diff
@@ -0,0 +1,36 @@
+
+[[run_diff]]
+= Identify Changed Configuration Files
+
+During an upgrade, {page-component-title} overwrites configuration files, so any customization you may have done will be lost.
+You need to back up your customized configuration files so that you can merge them into your upgraded system.
+
+Consider one of the following options to identify changed configuration files:
+
+* <<config-diff, run the config-diff.sh script>> (to show which files have changed)
+* make the `$OPENNMS_HOME/etc` folder a git repo to track changes between versions (to show content lines that have changed as well)
```

Comment:
I'm happy with this.  If they know git and want to use it, we've put the idea in their head.